extensions/big-data-cluster/src/bigDataCluster/dialog/addControllerDialog.ts

```diff
@@ -142,17 +187,35 @@ export class AddControllerDialog {
 		this.dialog.cancelButton.label = localize('textCancel', 'Cancel');
 	}
 
+	private get authValue(): AuthType {
+		return (<azdata.CategoryValue>this.authDropdown.value).name as AuthType;
+	}
+
+	private onAuthChanged(): void {
+		let isBasic = this.authValue === 'basic';
+		this.usernameInputBox.enabled = isBasic;
+		this.passwordInputBox.enabled = isBasic;
+		if (!isBasic) {
```

Comment:
Should probably disable the remember password checkbox too

Reply:
Done